assets/js/googlesitekit/modules/datastore/modules.js

```diff
@@ -206,11 +303,32 @@ const baseSelectors = {
 	 * @param {Object} state Data store's state.
 	 * @return {(Object|undefined)} Modules available on the site.
 	 */
-	getModules( state ) {
+	getModules: createRegistrySelector( ( select ) => ( state ) => {
 		const { modules } = state;
 
-		return modules;
-	},
+		// Return `undefined` if modules haven't been loaded yet.
+		if ( modules === undefined ) {
+			return undefined;
+		}
+
+		const registryKey = select( CORE_SITE ).getRegistryKey();
+		// Sorting the modules object by order property.
+		const sortedModules = sortObjectMapByKey( modules, 'order' );
```

Comment:
This is correct now, but when we change the function to work purely on arrays, we need to use `Object.values` here. Which makes sense anyway because below we need it in array format anyway.